host/xtest/regression_4000.c

```diff
@@ -5026,6 +5026,14 @@ static void xtest_tee_test_4011(ADBG_Case_t *c)
 			if (tmp[m] != 0xff)
 				break;
 		/* 4.2 */
+		if (!ADBG_EXPECT_COMPARE_UNSIGNED(c, n + i, < , sizeof(tmp)) ||
```

Comment:
I'd prefer if we fixed the code in a way a human (instead of a compiler) would understand.
That means checking right line 5019 that `tmp_size <= sizeof(tmp)`
The loop at line 5022 should only run to `tmp_size - i`
And so on.

Reply:
ok. I'll send a fix-up.